src/ModelStates/Transition.php

```diff
@@ -21,6 +21,26 @@ public function canTransition(): bool
         return true;
     }
 
+    public function direction(): TransitionDirection
+    {
+        return TransitionDirection::FORWARD();
+    }
+
+    public function isVisibleFromUserFrontend(): bool
+    {
+        return true;
+    }
```

Comment:
UserFrontend as opposed to AdminFrontend? Can you explain in a comment?

Reply:
Why not just `Frontend`? I was just confused by the term `UserFrontend`, as it implies there are multiple frontends.